packages/shared/lib/utils/error.ts

```diff
@@ -622,6 +622,21 @@ export class NangoError extends Error {
                 this.message = `The sync job results could not be updated: ${this.payload}`;
                 break;
 
+            case 'invalid_error':
+                this.status = 500;
+                this.message = `An invalid error of type: ${typeof this.payload}`;
+                break;
+
+            case 'http_error':
```

Comment:
should this error prefixed with `script_` since it only applies in the context of a script?